CMakeLists.txt

```diff
@@ -35,6 +35,13 @@ elseif (WIN32)
 	add_definitions (-DWIN32)
 endif()
 
+# compiler warning flags
+if (NOT WIN32)
+  SET( COMPILE_FLAGS "-Wall -Wno-sign-compare")
+  SET( CMAKE_CXX_FLAGS  "${COMPILE_FLAGS}" )
+  SET( CMAKE_C_FLAGS  "${COMPILE_FLAGS}" )
```

Comment:
We don't have any c code

Reply:
hmm, how about this one:
https://github.com/NCAR/VAPOR/blob/53b1fa78b25c1f84e1c1ca8680c8b849de71e79f/lib/vdc/kdtree.c

Reply:
Fair point